CXP-1168: Added Apps warning in events api basics and moved introduction (#595)
akeneo/pim-api-docs: content/events-api/overview.md

> # 👋 Welcome in the Events API basics documentation!

Reviewer: welcome to* ?

You are probably asking yourself a ton of questions about our Events API, that's why we made this article.

::: warning

> The Events API feature is only available in the **5.0 version** and higher and for **SaaS** customers.

Reviewer 2: ```suggestion
The Events API feature is only available since the **5.0 version** and for **SaaS** customers.
Please note that the events API feature is not currently available for **Apps**.
```

:::

> _All the essential things you need to know._

Reviewer 2: ```suggestion
# Overview

_All the essential things you need to know._
```

Author: done

Simple products and product variants have similar event formats, but events related to product models are a bit different. That is why we detailed each event type format in our [Events API Reference](../events-reference/events-reference-serenity/products.html)

> ::: warning

Reviewer 2: ```suggestion
```

> The events API feature is not currently available for **Apps**.

Reviewer 2: ```suggestion
```

> :::

Reviewer 2: ```suggestion
```
